include/deal.II/base/iterator_range.h

```diff
@@ -130,6 +132,11 @@ class IteratorRange
      */
     using BaseIterator = Iterator;
 
+    /**
+     * Typedef to the accessor type of the underlying iterator.
+     */
+    using AccessorType = typename BaseIterator::AccessorType;
```

Comment:
I don't think you can do this -- it will work for cell iterators, but in general, iterator classes don't have such a type.

Reply:
I came to the conclusion that this class must have been only designed to work on cell iterators (due to the way iterators-over-iterators works, if I recall correctly).

Reply:
I added this line because I needed to (something in `mesh_loop()` asks for this information, if I recall correctly)

Reply:
OK. I think you're probably correct that it is only used in this context.

Reply:
I managed to get rid of this :-)